api/fetch_tags.py

```
@@ -0,0 +1,84 @@
+                            # Match found labels against our defined tech keywords
+                            for tech in tech_keywords:
+                                if tech in label_name:
+                                    detected_tags.add(tech)
+            
+            # 5. Serve the clean payload transformed back into standard JSON format
+            return {
+                "count": total_count,
+                "tags": list(detected_tags)
+            }
+            
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)